ansible/roles/operator-pipeline/templates/openshift/tasks/digest-pinning.yml

```diff
@@ -82,8 +82,10 @@ spec:
         else
           echo "Manifests are pinned."
           echo -n "false" | tee $(results.dirty_flag.path)
+          echo "The branch is reset to its pre-pin state because other unrelated changes may occur during the pin step and need to be undone."
+          git checkout -- .
         fi
-    - name: verify-related-images
+    - name: verify-related-imagesmultiline echo
```

Comment:
This looks like an unwanted modification.

Reply:
yes, thanks!